torch/distributed/_sharded_tensor/__init__.py

```diff
@@ -101,7 +100,58 @@ def ones(
     Returns:
         A :class:`ShardedTensor` object on each rank
     """
-    tensor_init_params = TensorInitParams(create_op=CreateOp.ONES, dtype=dtype, layout=layout,
+    tensor_init_params = TensorInitParams(create_op=CreateOp.ONES, fill_value=0,
```

Comment:
Same comment as above, can we omit fill_value? Also we are creating ones here so fill_value=0 doesn't seem right.

Reply:
Great point, removed.